File: examples/java/basic_usage.java
#!/usr/bin/env java
/**
 * Basic usage example for HiTechCloud Tools Java SDK
 * 
 * Set your API key:
 *   export HITECHCLOUD_TOOLS_API_KEY=your-api-key
 */
import vn.hitechcloud.tools.HiTechCloudTools;

HiTechCloudTools client = new HiTechCloudTools("your-api-key");

// GeoIP lookup
var result = client.mangAndHaTang().geoipLookup("8.8.8.8");
System.out.println(result);

// Domain WHOIS
var whois = client.tenMienAndSsl().domainWhois("example.com");
System.out.println(whois);

File: examples/go/basic_usage.go
#!/usr/bin/env go
/**
 * Basic usage example for HiTechCloud Tools Go SDK
 * 
 * Set your API key:
 *   export HITECHCLOUD_TOOLS_API_KEY=your-api-key
 */
package main

import (
    "github.com/hitechcloud-vietnam/hitechcloud-tools-sdk/sdk/go"
)

func main() {
    client := hitechcloudtools.NewClient("your-api-key")
}

// GeoIP lookup
result, err := client.MangAndHaTang.GeoipLookup("8.8.8.8")
if err != nil {
    log.Fatal(err)
}
fmt.Println(result)

// Domain WHOIS
whois, err := client.TenMienAndSsl.DomainWhois("example.com")
if err != nil {
    log.Fatal(err)
}
fmt.Println(whois)
